### PR TITLE
added html form

### PR DESCRIPTION
**Pull request by Vitalii-Fediaiev**
[DEMO LINK](https://vitalii-fediaiev.github.io/layout_html-form/)
[TEST REPORT LINK](https://vitalii-fediaiev.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,9 @@
 /* styles go here */
+.form_field{
+  display: block;
+  margin: 10px 0;
+}
+
+.form-field{
+margin-bottom: 20px;
+}
```
Comment (liliia-livandovskaya) [inline src/style.css:9]: add one empty field in the end

File: src/style.css [hunk 2]
```diff
@@ -1 +1,9 @@
 /* styles go here */
```
Comment (liliia-livandovskaya) [inline src/style.css:1]: remove comments

File: src/index.html [hunk 3]
```diff
@@ -13,5 +13,152 @@
...
+
+        <label class="form_field">
+            Comments:
+
+          <textarea
+             required 
+             name="feedback"
+             minlength="1"
+             maxlength="5">
+          </textarea>
```
Comment (liliia-livandovskaya) [inline src/index.html:145]:
```suggestion
          <textarea
           required 
           name="feedback"
           minlength="1"
           maxlength="5">
          </textarea>
```

File: src/index.html [hunk 4]
```diff
@@ -13,5 +13,152 @@
...
+             type="color" 
+             name="color">
+        </div>
+
+        <div class="form_field">
+          <label> What time do you go to bed?</label>
+
+          <input
+             type="time" 
+             name="time">
```
Comment (liliia-livandovskaya) [inline src/index.html:111]:
```suggestion
          <input
             type="time" 
             name="time"
           >
```
fix code style all over the page

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
GJ! Pay attention to comments and checklist

File: src/style.css [hunk 5]
```diff
@@ -1 +1,9 @@
-/* styles go here */
+
+.form_field {
+  display: block;
+  margin: 10px 0;
+}
+
+.form-field {
+  margin-bottom: 20px;
+}
```
Comment (liliia-livandovskaya) [inline src/style.css:9]: add one more empty line in the end

File: src/index.html [hunk 6]
```diff
@@ -11,7 +11,213 @@
...
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+        <div class="form_field">
+          Surname:
+          
+          <label>
+            <input 
+             type="text"
+             autocomplete="off"
+
```
Comment (liliia-livandovskaya) [inline src/index.html:27]:
```suggestion
```

File: src/index.html [hunk 7]
```diff
@@ -11,7 +11,213 @@
...
+        </div>
+         
+        <div class="form_field">
+          Name:
+
+          <label>
+            <input 
+             type="text"
+             autocomplete="off"
+
```
Comment (liliia-livandovskaya) [inline src/index.html:39]: add name atribute

File: src/index.html (hunk 6 above)
Comment (liliia-livandovskaya) [reply src/index.html:27]: add name attribute

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
GJ! Pay attention to the comments

File: src/index.html [hunk 8]
```diff
@@ -11,7 +11,215 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="1" method="post">
```
Comment (Anastasiia-Svintsova) [inline src/index.html:17]: Add a correct action to the form. As the task says: 
On form submit send form data to https://mate-academy-form-lesson.herokuapp.com/create-application

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 8 above)
Comment (Vitalii-Fediaiev) [reply src/index.html:17]: I don't understand!
do i have to create a server? or write any fictional one?

File: src/index.html [hunk 9]
```diff
@@ -11,7 +11,200 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="http://localhost:8080/api" method="post">
```
Comment (Anastasiia-Svintsova) [inline src/index.html:17]:
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
     >
```

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Please, fix a one little detail

File: src/index.html [hunk 10]
```diff
@@ -11,7 +11,203 @@
...
+        <label class="form_field">
+            Comments:
+
+          <textarea
+           required 
+           name="feedback"
+           minlength="1"
+           maxlength="5"
+           >
+          </textarea>
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:194]:
```suggestion
          <textarea
            required 
            name="feedback"
            minlength="1"
            maxlength="5"
        ></textarea>
```

File: src/index.html [hunk 11]
```diff
@@ -11,7 +11,203 @@
...
+        </label>
+
+        <label class="form_field">
+           Would you recommend us?
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:198]:
```suggestion
         Would you recommend us?
```

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Good job - fix your code style and fix test - smth went wrong 🙃 
pay attention to the top margin of first input in fieldset and bottom margin of the last input in fieldset

**Pull request closed without merging**